bigquery/README.md

```diff
@@ -39,21 +41,48 @@ if you haven't done so already.
 
 ### Overview
 
-Index BigQuery tables into Elasticsearch. Only specified facet fields will be indexed.
+A Data explorer UI allows for faceted search. For example,
```

Comment:
Instructions above need an update:

    pip install requirements.txt

needs to be:

    pip install -r requirements.txt

Reply:
It's not needed because of the way requirements.txt is generated (see below), but I added it to avoid confusion.

Reply:
This is what I get without the -r:

```
$ pip install requirements.txt
Collecting requirements.txt
  Could not find a version that satisfies the requirement requirements.txt (from versions: )
No matching distribution found for requirements.txt
```